proto/pocket/migration/event.proto

```diff
@@ -108,4 +108,20 @@ message EventMorseSupplierClaimed {
   // Supplier staked as a result of the claim
   // - Mirrors Morse supplier stake
   shared.Supplier supplier = 5 [(gogoproto.jsontag) = "supplier"];
-}
\ No newline at end of file
+}
+
+// EventMorseAccountRecovered
+// - Emitted when a Morse account is recovered on-chain
+message EventMorseAccountRecovered {
+  // The session end height (on Shannon) in which the recovery was committed (i.e. claimed).
+  int64 session_end_height = 1 [(gogoproto.jsontag) = "session_end_height"];
+
+  // The total balance which was recovered.
```

Comment:
1. Does this include both unstaked and staked balance?
2. Does it auto-liquidate both of them at once?

If so, #PUC